lib/rubocop/cop/rspec/context_wording.rb

```diff
@@ -52,7 +53,7 @@ def on_block(node)
         private
 
         def bad_prefix?(description)
-          !prefixes.include?(description.split(/\b/).first)
+          !prefix_regex.match?(description)
```

Comment:
Maybe `prefixes.none? { |prefix| description.start_with?(prefix) }`?

Reply:
That's what I did at first, but it could then match part of a word, which would cause a failure in the following test case:
https://github.com/rubocop/rubocop-rspec/blob/1d957cc2c00492acd649073aa9f5e10b07c4a4ed/spec/rubocop/cop/rspec/context_wording_spec.rb#L43-L49

Reply:
Got it 👍 
`prefixes.none? { |prefix| description.start_with?("#{prefix} ") }` then?

Reply:
That would fail when the prefix is followed by a character other than space, such as the following test case:

https://github.com/rubocop/rubocop-rspec/blob/1d957cc2c00492acd649073aa9f5e10b07c4a4ed/spec/rubocop/cop/rspec/context_wording_spec.rb#L36-L41

Reply:
I guess there's no way around regexp then 😄

Reply:
start_with? also accepts regexp, as well as multiple arguments, so I was just wondering as well, why we don't use it

Reply:
@Darhazer `start_with?` accepts regular expressions in Ruby 2.5, but not in Ruby 2.4.